### ISSUE TITLE
display check_name for errors

### ISSUE THREAD

**Issue by mockdeep** [djezzzl/database_consistency#87]
I know setting `DEBUG` displays check names, but it would be nice if I could see only errors along with the check name. One thing I'm trying to do is selectively disable the checkers that are failing and gradually re-enable them as we correct issues in our schema. It's a little challenging to tell *which* checker is failing, though, so it's hard to know which one to disable.

**Comment by djezzzl** [comment]
Hi @mockdeep, thank you for using the gem!

Could you please show by examples what do you mean as I'm not sure I understood the issue?

**Comment by mockdeep** [comment]
@djezzzl I'm hoping to set up `database_consistency` to run on CI, but there are almost 100 violations currently. I would like to disable the checks that are failing for now and then we can gradually re-enable them. For example, here's a handful of messages:

```
fail Organization slugs associated model should have proper index in the database
fail Organization approver_lists associated model key (organization_id) with type (bigint) mismatches key (id) with type (integer)
fail Campaign user_id column should be required in the database
fail Group name column should be required in the database
fail User email column is required but there is possible null value insert
fail Doc fields_version column should be required in the database
fail Form user_id column should be required in the database
fail Form subject_id column should be required in the database
```

I would like to disable the checks associated with these failures, but I don't know the names. It would be nice if the output instead looked something like:

```
fail Organization slugs associated model should have proper index in the database (checker: MissingIndexChecker)
fail Organization approver_lists associated model key (organization_id) with type (bigint) mismatches key (id) with type (integer) (checker: ForeignKeyTypeChecker)
fail Campaign user_id column should be required in the database (checker: ColumnPresenceChecker)
fail Group name column should be required in the database (checker: ColumnPresenceChecker)
fail User email column is required but there is possible null value insert (checker: ColumnPresenceChecker)
fail Doc fields_version column should be required in the database (checker: ColumnPresenceChecker)
fail Form user_id column should be required in the database (checker: ColumnPresenceChecker)
fail Form subject_id column should be required in the database (checker: ColumnPresenceChecker)
```

That way I know which checker to disable. I know `DEBUG` provides this, but it also adds a lot more output that makes it hard to find on a larger schema.

**Comment by djezzzl** [comment]
I see. I think we can change that 👍

**Comment by djezzzl** [comment]
Hey 👋 ,

I've just released `1.1.2`. Could you please try it out and say if that helps you? Feel free to reopen the issue if necessary.
